lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -825,7 +825,7 @@ def test_grades_csv(self):
         headers = rows[0]
 
         # picking first student records
-        data = dict(zip(headers.strip().split(','), rows[1].strip().split(',')))
+        data = dict(zip(headers.strip().split(','), rows[2].strip().split(',')))
```

Comment:
So, this test originally failed on Jenkins.  I tested it locally, and it failed.  There's two rows of data the test has, and it picks the first one.  It looked like, somehow, the order of the rows changed, and it simply needed to look at the other row.

After making said change, the test ran successfully in my devstack.  It still fails on Jenkins, though, for reasons unbeknownst to me. :|